content/_data/upgrades/2-332-1.adoc

```diff
@@ -13,6 +13,16 @@ $ systemctl edit jenkins
 ----
 
 See the link:/doc/book/system-administration/systemd-services/[Managing systemd services] page for more details of `systemd` administration.
+See the link:/2022/03/25/systemd-migration/[blog post] for more details about the migration.
+
+If you find a regression, please file a bug report in link:https://issues.jenkins.io/[Jira].
+When reporting an issue, include the following information:
+
+. Use the `core` component.
```

Comment:
It is because the `packaging` component in Jira has been archived, and my request to reverse this action in jenkins-infra/helpdesk#2819 was rejected as being unnecessary.